packages/mobile/src/screens/favorites-screen/useCollectionsScreenData.ts

```diff
@@ -117,38 +117,34 @@ export const useCollectionsScreenData = ({
         return []
       }
 
-      return fetchedCollectionIds.filter((collectionId) => {
```

Comment:
Before the library work, `fetchedCollectionIds` was always populated even when offline bc we were using the collections value in the account object. Now, `fetchedCollectionIds` always comes from API (since we can't rely on the account object collections, since they only include the user's Favorites), so it'll be empty if we start the session offline. So to fix the issue we can just gather the downloaded collection IDS from the `offlineCollectionStatus` data in the store.

Reply:
Love this change and makes a ton of sense